extraTS/utils/Permissions.ts

```diff
@@ -0,0 +1,27 @@
+export default function hasPermission(moduleName,role:string,permissionType:string) {
+    let type;
+    const { all, read, write, Delete } = moduleName;
+    if (permissionType == 'all')
+        type = all;
+    if (permissionType == 'read')
+        type = read;
+    if (permissionType == 'write')
+        type = write;
+    if (permissionType == 'Delete')
+        type = Delete;
+
+    if (role == 'head-trainer') {
+        return true;
+    }
+    else {
+        if (type.includes(role))
+            return true
+        else
+            return false;
+    }
+}
+// const { getUsers, getUser } = permissions;
+// let result = hasPermission(getUsers, 'head-trainer', 'Delete');
+// let result_1 = hasPermission(getUsers, 'trainer', 'all');
+// console.log(result);
```

Comment:
Create the function as per instruction in task

Reply:
I make the function as per given in the task but by mistaken make wrong the folder. I resolved it